lib/recipe.rb

```diff
@@ -175,13 +178,114 @@
   notifies :restart, "service[#{agent_service_name}]" unless platform?('windows')
 end
 
+#
+# Windows platform
+#
 if platform?('windows')
-
   # In windows we run things as as a scheduled task instead of a daemon
   # This avoids the code overhead of a service, and isn't too inefficient since we run infrequently
+  # However this ends up essentially vendoring parts of the windows cookbook.
+  # https://github.com/chef-cookbooks/windows/blob/master/resources/task.rb
+  #
+  task_name = 'Chef Liveness Agent'
```

Comment:
We called it `automate-liveness-agent` or something similar on all the other platforms. Perhaps we should call it `Automate Liveness Agent` here for consistency?